### ISSUE TITLE
EmulationStation Crash

### ISSUE THREAD

**Issue by manurafal** [Aloshi/EmulationStation#914]
**System:** Windows 10 Pro with Retrobat 6.4

**CPU:** Intel Core i5 2400

**Hard Disc:** WD 4TB NTFS

**Memory:** 16GB

**Grafic Card:** Nvidia Gforce GT 1030 LP 2GB DDR5

**Problem:** 

When I move through the menu, Retrobat suddenly closes randomly, it happens randomly with any system. The error that appears in the **_es_log.txt_** file is the following:

_**2024-08-29 16:54:09	ERROR	Error - Failed to load image from memory!
2024-08-29 16:54:09	ERROR	Could not initialize texture from memory, invalid data!  (file path: D:/RetroBat/emulationstation/.emulationstation/themes/es-Hypermax_Plus_v9/frogger/char.png, data ptr: 71,004,192, reported size: 1,262,596)**_

I have checked the PNG file and it is correct. The error is random, sometimes it is a PNG, other times it is an MP4, or an OGG, or an MP3, and I have always checked the files and they are correct.

**Comment by udance4ever** [comment]
you may wish to file your issue with RetroBat directly:

https://github.com/RetroBat-Official/retrobat-setup

and close the issue here.

the RetroBat developers are very responsive!

**Comment by manurafal** [comment]
> you may wish to file your issue with RetroBat directly:
> 
> https://github.com/RetroBat-Official/retrobat-setup
> 
> and close the issue here.
> 
> the RetroBat developers are very responsive!

They don't know what it could mean, since it is something from ES.

**Comment by udance4ever** [comment]
it doesn’t hurt to try - you haven’t gotten a response here for 2 weeks.

RetroBat shares the same codebase with Batocera Linux - an enhanced fork of emulation station:

https://github.com/RetroBat-Official/emulationstation

**Comment by manurafal** [comment]
> it doesn’t hurt to try - you haven’t gotten a response here for 2 weeks.
> 
> RetroBat shares the same codebase with Batocera Linux - an enhanced fork of emulation station:
> 
> https://github.com/RetroBat-Official/emulationstation

I already made the query in Retrobat before doing it here, and the solution was to change the theme, although there are other people who have the same thing happen to me and it is not because of the theme. Asking here is in case you could give us some tips to follow to see if we can locate the error. It never hurts to have another point of view.

**Comment by udance4ever** [comment]
ah - thanks for clarifying that.  Did you try to change the theme?  Do you get the same error across multiple themes?

**Comment by manurafal** [comment]
> ah - thanks for clarifying that. Did you try to change the theme? Do you get the same error across multiple themes?

Sorry for my late response, but I'm very busy with work. 

Regarding your questions, I have tried other themes and have obtained the same error. As soon as I can, I will try again and send you the complete log file so you can see the error. Thank you very much for your help, and sorry for my delay.

**Comment by manurafal** [comment]
@udance4ever Sorry for my delay, I am attaching the file with the error. By the way, I now have version 6.4.0 of Retrobat, but I have the same error. I attach the es_log file obtained when it fails.
[es_log.txt](https://github.com/user-attachments/files/17606885/es_log.txt)

[es_log.zip](https://github.com/user-attachments/files/17606899/es_log.zip)

**Comment by manurafal** [comment]
[es_log.txt](https://github.com/user-attachments/files/17787992/es_log.txt)

![photo_2024-11-17_13-46-18](https://github.com/user-attachments/assets/aaf5f9ad-d5b0-410a-bd5b-45ffb3fb6db5)


I attach another log file and windows registry error that I have obtained using another theme, and emulationstation continues to close.

Please, I'm desperate and I don't know what to do anymore. Can you help me?

**Comment by manurafal** [comment]
![-5915836547503081186_121](https://github.com/user-attachments/assets/39ae36f6-b2b6-4e4b-949d-22898a98695f)
![-5915836547503081187_121](https://github.com/user-attachments/assets/10a1e902-0aaa-4a3f-a068-b18878f69854)
![-5915836547503081184_121](https://github.com/user-attachments/assets/b54453e6-fe75-42f3-98db-1c09bb8dadac)
![-5915836547503081185_121](https://github.com/user-attachments/assets/7c69aaea-9cd4-4eb9-84e5-11644639ae99)
![-5915836547503081188_121](https://github.com/user-attachments/assets/75f9d8dc-fa3a-44e4-96e2-0bc6a0bef3c2)

**Comment by manurafal** [comment]
Here I show you the errors that appear in the Windows event viewer.

**Comment by manurafal** [comment]
I finally found out what all my mistakes were due to. And it was due to Retrobat's VRAM option, in which I had 256MB set. But when I went up thinking that my mistake would be solved, the opposite happened, that is, it got worse. So now I have a value of 100mb and everything works correctly, it no longer gives me the error.
I thought that the higher that value was the better, but it turned out to be worse.

![Screenshot_1](https://github.com/user-attachments/assets/d9c6131a-3930-42fb-a3d1-4305d5cc1fc5)
